index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Sebastián's Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="flex-nav"> 
+            <a href="#" class="toggleNav">☰</a>
+        </nav>
+    </header>
+    <main class="name">
+        <h1>Sebastián<br>Muñoz</h1>
+    </main>
+    <section class="intro">
+        <h2>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br>
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </h2>
```

Comment:
Replace h2 with p as this is a paragraph and make sure the text is properly aligned on the page